<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GaussianBeamWindow</class>
 <widget class="QMainWindow" name="GaussianBeamWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="GaussianBeam.qrc">
     <normaloff>:/images/document-save.png</normaloff>:/images/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="icon">
    <iconset resource="GaussianBeam.qrc">
     <normaloff>:/images/document-save-as.png</normaloff>:/images/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="icon">
    <iconset resource="GaussianBeam.qrc">
     <normaloff>:/images/document-print.png</normaloff>:/images/document-print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="GaussianBeam.qrc">
     <normaloff>:/images/document-open.png</normaloff>:/images/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_AddLens">
   <property name="text">
    <string>&amp;Lens</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_AddFlatInterface">
   <property name="text">
    <string>&amp;Flat Interface</string>
   </property>
  </action>
  <action name="action_AddCurvedInterface">
   <property name="text">
    <string>&amp;Cuved Interface</string>
   </property>
  </action>
  <action name="action_AddFlatMirror">
   <property name="text">
    <string>Flat &amp;Mirror</string>
   </property>
  </action>
  <action name="action_AddCurvedMirror">
   <property name="text">
    <string>Cu&amp;rved Mirror</string>
   </property>
  </action>
  <action name="action_AddGenericABCD">
   <property name="text">
    <string>Generic &amp;ABCD</string>
   </property>
  </action>
  <action name="action_AddOptics">
   <property name="icon">
    <iconset resource="GaussianBeam.qrc">
     <normaloff>:/images/edit-add.png</normaloff>:/images/edit-add.png</iconset>
   </property>
   <property name="text">
    <string>Add optics</string>
   </property>
  </action>
  <action name="action_RemoveOptics">
   <property name="icon">
    <iconset resource="GaussianBeam.qrc">
     <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="action_AddDielectricSlab">
   <property name="text">
    <string>&amp;Dielectric slab</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset resource="GaussianBeam.qrc">
     <normaloff>:/images/document-new.png</normaloff>:/images/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="GaussianBeam.qrc"/>
 </resources>
 <connections/>
</ui>
